api/test/pact/pact-tests/wa/assignTask.spec.ts

```diff
@@ -0,0 +1,61 @@
+import { expect } from 'chai';
+import { PactTestSetup } from '../settings/provider.mock';
+import { assignTaskToUser } from "../../pactUtil";
+
+const { Matchers } = require('@pact-foundation/pact');
+const { somethingLike } = Matchers;
+const pactSetUp = new PactTestSetup({ provider: 'wa_task_management_api_assign_task_by_id', port: 8000 });
+
+
+describe("Task management api, assign a task to user", () => {
+
+    const mockRequest = {
+        user_id: "004b7164-0943-41b5-95fc-39794af4a9fe"
+    }
+    const taskId = "12345678";
+    describe("post /task/taskId/assign", () => {
+
+        const jwt = 'some-access-token';
+
+        before(async () => {
+            await pactSetUp.provider.setup()
+            const interaction = {
+                state: "some tasks exists on event for case",
```

Comment:
the provider state for this is "assign a task using taskId"

Reply:
updated